app/models/project.rb

```diff
@@ -74,7 +74,10 @@ def siblings
   end
 
   def clone_stories_into(clone)
-    stories.each { |story| clone.stories.create(story.dup.attributes) }
+    stories.each do |story|
+      attrs = story.dup.attributes.except("status", "id", "created_at", "updated_at")
```

Comment:
JFI: we do not need to add the `id`, `created_at`, `updated_at` the `dup` method removed them already

```suggestion
      attrs = story.dup.attributes.except(:status)
```

In the other hand, I was wondering if instead of `except` attributes we would want to accept only the "clonable" attributes we need, I'm thinking towards the future when someone adds more attributes those will be cloned and maybe we don't want that, makes sense?